string node_name
string msg_name
---
int16 id
